Marlin/Configuration_adv.h

```diff
@@ -3027,6 +3031,11 @@
   #endif
 #endif
 
+#ifdef SPINDLE_SERVO
+  #define SPINDLE_SERVO_ID 0
+  #define SPINDLE_SERVO_MIN 10
+#endif
```

Comment:
I think these should be directly under `SPINDLE_SERVO`, like this:

```
//#define SPINDLE_SERVO // If you are using RC ESC to drive spindle motor. Remember to enable servos and disable SPINDLE_LASER_FREQUENCY
#if ENABLED(SPINDLE_SERVO)
  #define SPINDLE_SERVO_ID 0
  #define SPINDLE_SERVO_MIN 10
#endif
```

The `MIN` setting could probably use a comment also. I don't know whether that is the lowest active setting, or if that defines "off".

Reply:
Maybe change the name for the MIN value so that it is clear whether the input is degrees or microseconds.

I thought the servo commands accepted both (and assumed < 360 was an angle). Now that I go look at the code I am less sure of that (or maybe it depends on the HAL).

Reply:
Maybe you should just be using the existing constants, and rework the config file so it can work for both Servo and PWM spindles:

```
    #define SPEED_POWER_INTERCEPT         0    // (%) 0-100 i.e., Minimum power percentage
    #define SPEED_POWER_MIN               0    // (%) 0-100
    #define SPEED_POWER_MAX             100    // (%) 0-100
    #define SPEED_POWER_STARTUP          80    // (%) M3/M4 speed/p
```